resources/js/features/game-list/utils/column-definitions/buildHasActiveOrInReviewClaimsColumnDef.tsx

```diff
@@ -47,18 +44,22 @@ export function buildHasActiveOrInReviewClaimsColumnDef({
 
       return (
         <div>
-          {hasActiveOrInReviewClaims ? (
-            <BaseTooltip>
-              <BaseTooltipTrigger asChild>
-                <p>{strings.t_yes}</p>
-              </BaseTooltipTrigger>
+          {hasActiveOrInReviewClaims && row.original.game.claimants ? (
+            <div className="flex items-center gap-1.5">
+              <LuCheck className="size-4" />
+              <span className="sr-only">{strings.t_yes}</span>
```

Comment:
I find the X misleading, especially right next to the add-to-list column
![image](https://github.com/user-attachments/assets/3df4f19f-0ccc-452e-88b8-fdfed29bc2f2)

Similarly, the check mark is redundant. Either there will be one or more avatars, or there will be zero avatars. Do we really need the X/Check too?